datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaBasedConnectorTask.java

```diff
@@ -767,9 +773,13 @@ protected void updateConsumerAssignment(Collection<TopicPartition> partitions) {
 
   @Override
   public void onPartitionsRevoked(Collection<TopicPartition> topicPartitions) {
+    if (_skipOnPartitionsRevoked) {
+      _logger.warn("Skipping commit in onPartitionsRevoked due to an exception.");
```

Comment:
Can you plz fix the warning ? The commit is skipping because of consumer close, which is regular shutdown case as well. Also, this is not really a warning. You can probably change this to "INFO".